pysat/tests/test_utils_files.py

```diff
@@ -0,0 +1,45 @@
+import numpy as np
+from pysat.utils import files as futils
+
+
+class TestBasics():
+
+    temporary_file_list = False
+
+    def setup(self):
+        """Runs before every method to create a clean testing setup."""
+
+    def teardown(self):
+        """Runs after every method to clean up previous testing."""
+
+    def test_parse_delimited_filename(self):
+        """Check ability to parse delimited files"""
+        # Note: Can be removed if future instrument that uses delimited
```

Comment:
Instead of having a for-loop in this test, I recommend parametrizing it.

Reply:
resolved in conversation